<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSVNUpdateToRevisionDialog</class>
 <widget class="QDialog" name="QSVNUpdateToRevisionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="revision_groupBox">
     <property name="title">
      <string>Revision</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
      <item row="0" column="0">
       <widget class="QRadioButton" name="head_radioButton">
        <property name="text">
         <string>HEAD revision</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="showlog_pushButton">
        <property name="text">
         <string>Show log</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="revision_radioButton">
        <property name="text">
         <string>Revision</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="revision_lineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="depth_groupBox">
     <property name="title">
      <string>Update Depth</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="depth_comboBox">
        <item>
         <property name="text">
          <string>Working copy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fully recursive</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Immediate children, including folders</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Only file children</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Only this item</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Exclude</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="items_pushButton">
        <property name="text">
         <string>Choose items...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="sticky_checkBox">
        <property name="text">
         <string>Make depth sticky</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="misc_groupBox">
     <property name="title">
      <string>Misc</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="omit_checkBox">
        <property name="text">
         <string>Omit externals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>head_radioButton</tabstop>
  <tabstop>revision_radioButton</tabstop>
  <tabstop>showlog_pushButton</tabstop>
  <tabstop>revision_lineEdit</tabstop>
  <tabstop>depth_comboBox</tabstop>
  <tabstop>items_pushButton</tabstop>
  <tabstop>sticky_checkBox</tabstop>
  <tabstop>omit_checkBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QSVNUpdateToRevisionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QSVNUpdateToRevisionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
